light_node/src/main.rs

```diff
@@ -249,6 +253,8 @@ fn main() {
         context_storage::ContextPrimaryIndex::descriptor(),
         context_storage::ContextByContractIndex::descriptor(),
         SystemStorage::descriptor(),
+        SkipList::<String, Vec<u8>, HashMap<String, Vec<u8>>>::descriptor(),
```

Comment:
Use type alias instead of `Vec<u8>` to be more explicit about data stored in the hash map.

Reply:
Changed in 4b31638 to `<String, Bucket<Vec<u8>>, ContextMap>`. The `Vec<u8>` type is most explicit type we have, as we literally store *some* binary data, we do not have full encoding for.